web/static/jira/connect/index.html

```diff
@@ -97,20 +92,15 @@ <h1>Atlassian Jira - Create a New Connection</h1>
             Jira Data Center: Personal Access Tokens
           </a>
         </li>
-        <li>
-          <a href="" target="_blank" class="info">
-            TODO(ENG-965): Docusaurus guide, like GitHub
-          </a>
-        </li>
       </ul>
 
       <!-- TODO(ENG-799): remove ID once its unused -->
       <form id="save-form" method="post" action="/jira/save">
         <div class="form-group">
-          <label for="url">Base URL</label>
+          <label for="base_url">Base URL</label>
           <input
             type="url"
-            name="url"
+            name="base_url"
```

Comment:
Since this field has a placeholder, I would add a placeholder for the rest of the fields (email and token).